docs/production/deploying-to-cloud/secrets/AWS.mdx

```diff
@@ -1,29 +1,98 @@
 ---
 title: "AWS Secrets Manager"
 sidebarTitle: "AWS"
+description: "Attention Magers: This classified document contains vital intel on securing sensitive data within the AWS Secrets Manager vault. Study the protocols carefully to create and operationalize confidential secrets, granting you clearance to integrate them into your Mage data pipeline projects."
 ---
+![It's Classified](https://github.com/mage-ai/assets/blob/main/secrets_assets/Source_%20Giphy%20(4).gif?raw=True)
+### **Creating a New Secret in AWS Secrets Manager**
 
-### Python
+Before you can use AWS Secrets Manager with Mage, you need to create a new secret in the AWS Secrets Manager console. Follow these steps:
 
-To use secrets in Python code, read these [instructions](/design/data-loading#aws-secret-loader).
+1. Open the AWS Secrets Manager console: https://console.aws.amazon.com/secretsmanager/
+2. Click "Store a new secret."
+3. On the "Choose Secret Type" page, select the appropriate secret type for your use case, and then click "Next."
+4. On the "Configure Secret" page, provide the necessary details for your secret, such as the key-value pairs or the plaintext value, and then click "Next."
+5. (Optional) On the "Configure Rotation" page, you can enable automatic rotation for your secret. Automatic rotation periodically updates the secret value, ensuring that your sensitive data remains secure. Configure the rotation settings according to your requirements, and then click "Next."
+6. Review the secret details on the "Review" page, and if everything looks correct, click "Store."
 
-### YAML
+Your new secret is now stored in AWS Secrets Manager. Make a note of the secret's ARN (Amazon Resource Name), as you'll need it to access the secret from Mage.
+
+For more detailed information on managing and storing secrets in AWS Secrets Manager, refer to the official AWS documentation:
```

Comment:
Embed "AWS documentation" with the link using markdown.

Reply:
Added embedded link, tested and takes me to correct site from mintlify dev env.
<img width="440" alt="Screenshot 2024-06-20 at 1 22 12 PM" src="https://github.com/mage-ai/mage-ai/assets/92221031/4a81fa65-57b7-485b-91ed-34744d1f74ce">